lms/djangoapps/certificates/models.py

```diff
@@ -876,20 +893,58 @@ def is_enabled_for_course(cls, course_key):
         except cls.DoesNotExist:
             return False
         else:
-            return latest.enabled
+            return latest.self_generation_enabled or latest.enabled
```

Comment:
What if someone explicitly sets `self_generation_enabled` to FALSE after the migration? If it had previously been set to TRUE, this logic would continue to return TRUE. A better approach would be to check if `self_generation_enabled` is set (not-null), and if so return it. Otherwise return `enabled`.

Reply:
Alternative solution: stop using `enabled`. The data will be in place for `self_generation_enabled` before the code using the column is in place. With that assurance, you can safely replace usage of `enabled` with `self_generation_enabled`.